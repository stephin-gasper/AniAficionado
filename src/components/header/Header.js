import React from 'react';
import Logo from '../logo';
import {HeaderWrapper, LogoWrapper} from './Header.style';

const Header = () => (
  <HeaderWrapper>
    <LogoWrapper>
      <Logo />
    </LogoWrapper>
  </HeaderWrapper>
);

export default Header;
