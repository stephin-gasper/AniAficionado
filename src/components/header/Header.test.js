import React from 'react';
import {View as MockView} from 'react-native';
import {render} from 'test/test-utils';

import Header from './Header';

jest.mock('../logo', () => (props) => <MockView testID="logo" {...props} />);

describe('<Header />', () => {
  it('should render correctly', () => {
    expect(render(<Header />)).toMatchSnapshot();
  });
});
