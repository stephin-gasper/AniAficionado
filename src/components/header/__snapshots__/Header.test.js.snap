// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#1e2226",
        "paddingBottom": 10,
        "paddingLeft": 20,
        "paddingRight": 20,
        "paddingTop": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
        },
      ]
    }
  >
    <View
      testID="logo"
    />
  </View>
</View>
`;
